### ISSUE TITLE
travis fixes

### ISSUE THREAD

**Issue by headupinclouds** [elucideye/drishti#614]
https://blog.travis-ci.com/2017-11-21-xcode8-3-default-image-announce

The Xcode 8.1 images have been removed.  Xcode 8.3 is the new default ~~~but we can update to 9.1 as well for longer lifespan~~~.  This will require updating polly toolchains.  As part of the same effort we can drop the `LTO` variants to speed things up.

UPDATE: Xcode 9 requires a Mac running macOS 10.12.6 or later, which I can't test locally.  We can use the default `osx_image: xcode8.3` image.

> Xcode 8.3.3 requires a Mac running macOS 10.12 or later.
> Xcode 8.3.3 includes SDKs for iOS 10.3.1, watchOS 3.2, macOS 10.12.4, and tvOS 10.2. 

Summary: Update to Xcode 8.3 SDK's and drop LTO 

Need: `osx-10-12-hid-sections`, `ios-nocodesign-10-3-arm64-dep-9-0-device-libcxx-hid-sections`

```
    - os: osx
      osx_image: xcode8.3      
      env: CONFIG=Release TOOLCHAIN=osx-10-12-hid-sections INSTALL=--install

    - os: osx
      osx_image: xcode8.3
      env: CONFIG=MinSizeRel TOOLCHAIN=ios-nocodesign-10-3-arm64-dep-9-0-device-libcxx-hid-sections INSTALL=--install

    - os: osx
      osx_image: xcode8.3
      env: CONFIG=Release TOOLCHAIN=osx-10-12-sanitize-address-hid-sections INSTALL=--install

    - os: osx
      osx_image: xcode8.3
      env: CONFIG=MinSizeRel TOOLCHAIN=android-ndk-r10e-api-19-armeabi-v7a-neon-hid-sections INSTALL=--strip
```

**Comment by ruslo** [comment]
Just for the info: https://github.com/ingenue/hunter/commit/46e418649e2ba11d28ad7afe6c2d12c19eab0ade

**Comment by headupinclouds** [comment]
I've upgraded to 8.3 and am testing: `osx-10-12-hid-sections` and `ios-nocodesign-10-3-arm64-dep-9-0-device-libcxx-hid-sections` now.  Maybe it makes sense to switch iOS builds to bitcode.  I don't know how compilation times are, but the deployed assets would probably be more general purpose with that toolchain.

**Comment by headupinclouds** [comment]
tested new toolchains and submitted PR's to polly
[ ] https://github.com/ruslo/polly/pull/206
[ ] https://github.com/ruslo/polly/pull/205
[ ] update drishti-upload/.travis.yml
[ ] update drishti/.travis.yml

**Comment by headupinclouds** [comment]
Merged #616
